pkg/machines/components/create-vm-dialog/autoDetectOS.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
```

Comment:
python3

Reply:
Does it actually work wity python2 this script as well?

Reply:
@KKoukiou, yes, both scripts work with python2 and python3

` ` `
fidencio@laerte ~/src/upstream/cockpit $ python2.7 pkg/machines/components/create-vm-dialog/autoDetectOS.py https://dl.fedoraproject.org/pub/fedora/linux/releases/29/Workstation/x86_64/os/
http://fedoraproject.org/fedora/29
fidencio@laerte ~/src/upstream/cockpit $ python2.7 pkg/machines/components/create-vm-dialog/autoDetectOS.py https://dl.fedoraproject.org/pub/fedora/linux/releases/29/Workstation/x86_64/os/images/boot.iso
http://fedoraproject.org/fedora/29
fidencio@laerte ~/src/upstream/cockpit $ python3.7 pkg/machines/components/create-vm-dialog/autoDetectOS.py https://dl.fedoraproject.org/pub/fedora/linux/releases/29/Workstation/x86_64/os/
http://fedoraproject.org/fedora/29
fidencio@laerte ~/src/upstream/cockpit $ python3.7 pkg/machines/components/create-vm-dialog/autoDetectOS.py https://dl.fedoraproject.org/pub/fedora/linux/releases/29/Workstation/x86_64/os/images/boot.iso
http://fedoraproject.org/fedora/29
fidencio@laerte ~/src/upstream/cockpit $ python2.7 pkg/machines/getOSList.py | wc -l
570
fidencio@laerte ~/src/upstream/cockpit $ python3.7 pkg/machines/getOSList.py | wc -l
570
```